rest/src/main/java/org/jboss/pnc/rest/endpoint/BuildConfigurationEndpoint.java

```diff
@@ -507,4 +507,22 @@ public Response getLatestBuildRecord(@ApiParam(value = "Build configuration id",
         return this.fromSingleton(buildRecordProvider.getLatestBuildRecord(id));
     }
 
+    @ApiOperation(value = "Get all BuildRecords (running and archived) associated with this Build Configuration, returns empty list if no build records are found")
```

Comment:
To be removed after testing, will be available via `pnc-rest/rest/builds?q=buildConfigurationAudited.idRev.id==1`